docs/en/api/render.md

```diff
@@ -0,0 +1,103 @@
+# `render(component {, options}])`
+
+- **Arguments:**
+
+  - `{Component} component`
+  - `{Object} options`
+    - `{Object} context`
+      - `{Array<Component|Object>|Component} children`
+    - `{Object} slots`
+        - `{Array<Componet|Object>|Component|String} default`
+        - `{Array<Componet|Object>|Component|String} named`
+    - `{Object} mocks`
+    - `{Object|Array<string>} stubs`
+    - `{Vue} localVue`
+
+- **Returns:** `{CheerioWrapper}`
+
+- **Options:**
+
+See [options](./options.md)
+
+- **Usage:**
+
+Renders an object to a string and returns a [cheerio wrapper](https://github.com/cheeriojs/cheerio).
+
+Cheerio is a node implementation of the jQuery library. It has a similar API to the Vue Test Utils [`Wrapper`](wrapper/README.md).
```

Comment:
IMHO, Since I think node is Node.js, I think this is better.
Cheerio is a library like jQuery to handle DOM in Node.js.

Reply:
Thanks, how about:

`Cheerio is an implementation of core jQuery designed for Node.js.` 
or 
`Cheerio is jQuery-like library to traverse the DOM in Node.js`

Reply:
> Cheerio is an implementation of core jQuery designed for Node.js.
has my vote.

Reply:
Since I think that the words which is `core jQuery` is not easy to understand,
the below sentence is better.
> Cheerio is jQuery-like library to traverse the DOM in Node.js

Reply:
I've gone with:

>Cheerio is jQuery-like library to traverse the DOM in Node.js

because it's easier to understand

Reply:
IMHO, there is the blank lines.
I think it is better that the blank lines removed and 2 space is added to end of line. 

```
Renders an object to a string and returns a [cheerio wrapper](https://github.com/cheeriojs/cheerio).  
Cheerio is a node implementation of the jQuery library. It has a similar API to the Vue Test Utils [`Wrapper`](wrapper/README.md).  
`render` uses [`vue-server-renderer`](https://ssr.vuejs.org/en/basic.html) under the hood, to render a component to static HTML.  
`render` is exported from the `@vue/server-test-utils` package.
```

Renders an object to a string and returns a [cheerio wrapper](https://github.com/cheeriojs/cheerio).  
Cheerio is a node implementation of the jQuery library. It has a similar API to the Vue Test Utils [`Wrapper`](wrapper/README.md).  
`render` uses [`vue-server-renderer`](https://ssr.vuejs.org/en/basic.html) under the hood, to render a component to static HTML.  
`render` is exported from the `@vue/server-test-utils` package.